zcash_client_sqlite/src/wallet.rs

```diff
@@ -2020,13 +2069,20 @@ pub(crate) fn update_expired_notes(
     conn: &rusqlite::Connection,
     expiry_height: BlockHeight,
 ) -> Result<(), SqliteClientError> {
-    let mut stmt_update_expired = conn.prepare_cached(
+    let mut stmt_update_sapling_expired = conn.prepare_cached(
         "UPDATE sapling_received_notes SET spent = NULL WHERE EXISTS (
             SELECT id_tx FROM transactions
             WHERE id_tx = sapling_received_notes.spent AND block IS NULL AND expiry_height < ?
         )",
     )?;
-    stmt_update_expired.execute([u32::from(expiry_height)])?;
+    stmt_update_sapling_expired.execute([u32::from(expiry_height)])?;
+    let mut stmt_update_orchard_expired = conn.prepare_cached(
```

Comment:
This technically isn't necessary unless the `orchard` feature is enabled. It is, however, safe regardless.